ironfish/src/network/peers/peer.ts

```diff
@@ -578,5 +596,18 @@ export class Peer {
     this.onStateChanged.clear()
     this.onKnownPeersChanged.clear()
     this.onMessage.clear()
+    this.onBanned.clear()
+  }
+
+  punish(score: number, reason?: string): boolean {
```

Comment:
Should this take BAN_SCORE or is it more flexible?

Reply:
I don't think so. ban score is a number, and BAN_SCORE is really just suggested relative ban score values you can use across the code base. I'd like to keep this flexible. I'm not really sure if I want to keep BAN_SCORE or not.